core/utils.js

```diff
@@ -38,6 +38,20 @@ goog.require('goog.events.BrowserFeature');
 goog.require('goog.math.Coordinate');
 goog.require('goog.userAgent');
 
+
+/**
+ * To allow ADVANCED_OPTIMIZATIONS, combining variable.name and variable['name']
+ * is not possible. To access the exported Blockly.Msg.Something it needs to be
+ * accessed through the exact name that was exported. Note, that all the exports
+ * are happening as the last thing in the generated js files, so they won't be
+ * accessible before JavaScript loads!
+ * @return {!Object<string, string>}
```

Comment:
To fix an eslint error (missing return description) make this

`@return {!Object<string, string>} The map of exported messages.`